sqlmesh/core/snapshot/definition.py

```diff
@@ -875,7 +875,14 @@ def _ensure_categorized(self) -> None:
 
 def table_name(physical_schema: str, name: str, version: str, is_temp: bool = False) -> str:
```

Comment:
shouldn't this take a catalog ?

Reply:
Well the solution is to take the catalog from the name if it is defined.

Reply:
Are you saying as a way to override it?

Reply:
ok nvm i got it now i think